docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+
+The New Ulm city zone is included within the Brown County zone. To avoid the "zone overlap constraint" issue, Heartland Express can be defined with four trips:
+
+- Service in New Ulm from 6:15 AM to 8:00 AM on weekdays.
+- County-wide service from 8:00 AM to 5:00 PM on weekdays.
+- Service in New Ulm from 5:00 PM to 5:45 PM on weekdays.
+- Service in New Ulm from 8:00 AM to 12:00 PM on Sundays.
+  
+[**trips.txt**](../../reference/#tripstxt)
+
+route_id | service_id | trip_id
+-- | -- | -- 
+74362 | c_67295_b_77497_d_31 | t_5374945_b_77497_tn_0
+74362 | c_67295_b_77497_d_31 | t_5374946_b_77497_tn_0
+74362 | c_67295_b_77497_d_31 | t_5374944_b_77497_tn_0
+74362 | c_67295_b_77497_d_64 | t_5374947_b_77497_tn_0
+
+`service_id = c_67295_b_77497_d_31` refers to weekdays, `service_id = c_67295_b_77497_d_64` refers to Sunday.
+
+### Define zones (GeoJSON locations)
+
+Using locations.geojson to define the operational zone of Heartland Express service, separate zones must be defined for Brown County and New Ulm City. Below is a simplified GeoJSON defining the zone of Brown County:
+```json
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "id": "area_708",
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        # Simplified, only presenting 3 coordinates here.
+        "coordinates": [
+          [
+            [
+              -94.7805702,
+              44.4560958
+            ],
+            [
+              -94.7805608,
+              44.4559928
+            ],
+            [
+              -94.7805218,
+              44.4559649
+            ]
+          ]
+        ]
+      },
+      "properties": {}
+    }
+  ]
+```
+
+### Define booking rules
+
+Here are booking rules that apply to all Heartland Express services:
+
+- Ride requests should be made between 8 AM and 3 PM weekdays. 
+- Rides must be requested one business day prior to day of the ride. 
+- Ride requests can be made up to 14 days in advance.
+
+Using `booking_type = 2` indicates that the service requires up to prior day(s) booking. `prior_notice_last_day = 1` and `prior_notice_start_day = 14` indicate  indicate that the service can be booked as early as 14 days in advance and as late as the day before.
+
+[**booking_rules.txt**](../../reference/#booking_rulestxt)
+
+booking_rule_id | booking_type | prior_notice_start_day | prior_notice_start_time | prior_notice_last_day | prior_notice_last_time | message | phone_number | info_url
+-- | -- | -- | -- | -- | -- | -- | -- | --
+booking_route_74362 | 2 | 14 | 8:00:00 | 1 | 15:00:00 | Brown County Heartland Express provides door-to-door on-demand transportation. To request a ride, call 1-507-359-2717 or 1-800-707-2717 by 3pm at least one business day ahead of your trip. | (507) 359-2717 | https://www.co.brown.mn.us/heartland-express-transit
+
+### Define stop_times
+
+- The operating hours are defined using the `start_pickup_drop_off_window` and `end_pickup_drop_off_window` fields.
```

Comment:
I think we can remove the first level of bullet point for consistency with other sections.

Reply:
Updated format